tfe/organization_members_helpers.go

```diff
@@ -0,0 +1,41 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+)
+
+func fetchOrganizationMembers(client *tfe.Client, orgName string, options *tfe.OrganizationMembershipListOptions) ([]map[string]string, []map[string]string, error) {
```

Comment:
I think `options` should be defined locally here rather than as a parameter. Suppose the caller of `fetchOrganizationMembers()` mutates `options` ... could lead to some wonky behavior here. Alternatively, if you do want the caller to be able to define `options`, it should be passed by value.

Reply:
right, to make it flexible for callers of this function to pass in respective options, I will update the option to pass by value.